### ISSUE TITLE
Crash with Compose 1.6

### ISSUE THREAD

**Issue by amv-bamboo** [BambooAppsDevTeam/Material3PullToRefresh#2]
After updating to Compose 1.6, we're getting the following exception:

```
FATAL EXCEPTION: main
Process: eu.bambooapps.material3.pullrefresh.sample, PID: 25453
java.lang.NoSuchMethodError: No virtual method at(Ljava/lang/Object;I)Landroidx/compose/animation/core/KeyframesSpec$KeyframeEntity; in class Landroidx/compose/animation/core/KeyframesSpec$KeyframesSpecConfig; or its super classes (declaration of 'androidx.compose.animation.core.KeyframesSpec$KeyframesSpecConfig' appears in /data/app/~~8rbHLZ1oNr1KiEAr0X6Uqw==/eu.bambooapps.material3.pullrefresh.sample-34ZFYjr9osj93hJ_omZOYw==/base.apk)
	at androidx.compose.material3.ProgressIndicatorKt$CircularProgressIndicator$endAngle$1.invoke(ProgressIndicator.kt:371)
	at androidx.compose.material3.ProgressIndicatorKt$CircularProgressIndicator$endAngle$1.invoke(ProgressIndicator.kt:369)
	at androidx.compose.animation.core.AnimationSpecKt.keyframes(AnimationSpec.kt:649)
	at androidx.compose.material3.ProgressIndicatorKt.CircularProgressIndicator-LxG7B9w(ProgressIndicator.kt:369)
	at eu.bambooapps.material3.pullrefresh.PullRefreshIndicatorKt$PullRefreshIndicator$1$1.invoke(PullRefreshIndicator.kt:129)
	at eu.bambooapps.material3.pullrefresh.PullRefreshIndicatorKt$PullRefreshIndicator$1$1.invoke(PullRefreshIndicator.kt:118)
	at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:118)
	at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:35)
	at androidx.compose.animation.CrossfadeKt$Crossfade$5$1.invoke(Crossfade.kt:133)
	at androidx.compose.animation.CrossfadeKt$Crossfade$5$1.invoke(Crossfade.kt:128)
	at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:109)
	at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:35)
	at androidx.compose.animation.CrossfadeKt.Crossfade(Crossfade.kt:142)
	at androidx.compose.animation.CrossfadeKt.Crossfade(Crossfade.kt:57)
	at eu.bambooapps.material3.pullrefresh.PullRefreshIndicatorKt$PullRefreshIndicator$1.invoke(PullRefreshIndicator.kt:114)
	at eu.bambooapps.material3.pullrefresh.PullRefreshIndicatorKt$PullRefreshIndicator$1.invoke(PullRefreshIndicator.kt:113)
	at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:109)
	at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:35)
	at androidx.compose.material3.SurfaceKt$Surface$1.invoke(Surface.kt:132)
	at androidx.compose.material3.SurfaceKt$Surface$1.invoke(Surface.kt:114)
	at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:109)
	at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:35)
	at androidx.compose.runtime.CompositionLocalKt.CompositionLocalProvider(CompositionLocal.kt:228)
	at androidx.compose.material3.SurfaceKt.Surface-T9BRK9s(Surface.kt:111)
	at eu.bambooapps.material3.pullrefresh.PullRefreshIndicatorKt.PullRefreshIndicator-3GLzNTs(PullRefreshIndicator.kt:96)
	at eu.bambooapps.material3.pullrefresh.sample.MainActivityKt.PullRefreshSample(MainActivity.kt:62)
	at eu.bambooapps.material3.pullrefresh.sample.ComposableSingletons$MainActivityKt$lambda-1$1.invoke(MainActivity.kt:36)
	at eu.bambooapps.material3.pullrefresh.sample.ComposableSingletons$MainActivityKt$lambda-1$1.invoke(MainActivity.kt:32)
	at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:109)
	at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:35)
	at androidx.compose.runtime.CompositionLocalKt.CompositionLocalProvider(CompositionLocal.kt:228)
	at androidx.compose.material3.TextKt.ProvideTextStyle(Text.kt:360)
	at androidx.compose.material3.MaterialThemeKt$MaterialTheme$1.invoke(MaterialTheme.kt:81)
	at androidx.compose.material3.MaterialThemeKt$MaterialTheme$1.invoke(MaterialTheme.kt:80)
	at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:109)

```

It seems to originate from `CircularProgressIndicator`. Need to investigate what's going on there

**Comment by lisonge** [comment]
same problem

---

It looks related to this

<https://github.com/JetBrains/compose-multiplatform/issues/4157>

**Comment by lisonge** [comment]
A simple repair method is to upgrade your dependence

```txt
androidx.compose.material3:material3:1.2.0-rc01
```

**Comment by amv-bamboo** [comment]
Fixed the issue with 1.1.0 release
